pkg/local_workflows/ignore_workflow/ignore_workflow.go

```diff
@@ -0,0 +1,333 @@
+
+func InitIgnoreWorkflows(engine workflow.Engine) error {
+	createFlagset := pflag.NewFlagSet(ignoreCreateWorkflowName, pflag.ExitOnError)
+	createFlagset.String(FindingsIdKey, "", findingsIdDescription)
+	createFlagset.String(IgnoreTypeKey, "", ignoreTypeDescription)
+	createFlagset.String(ReasonKey, "", reasonDescription)
+	createFlagset.String(ExpirationKey, "", expirationDescription)
+	createFlagset.String(RemoteRepoUrlKey, "", remoteRepoUrlDescription)
+	// If set to false, no response will be returned
+	createFlagset.Bool(EnrichResponseKey, false, "")
+	createFlagset.Bool(InteractiveKey, true, "")
+	_, err := engine.Register(WORKFLOWID_IGNORE_CREATE, workflow.ConfigurationOptionsFromFlagset(createFlagset), ignoreCreateWorkflowEntryPoint)
+	if err != nil {
+		return err
+	}
+
+	editFlagset := pflag.NewFlagSet(ignoreEditWorkflowName, pflag.ExitOnError)
+	editFlagset.String(IgnoreIdKey, "", ignoreIdDescription)
+	editFlagset.String(FindingsIdKey, "", findingsIdDescription)
+	editFlagset.String(IgnoreTypeKey, "", ignoreTypeDescription)
+	editFlagset.String(ReasonKey, "", reasonDescription)
+	editFlagset.String(ExpirationKey, "", expirationDescription)
+	editFlagset.String(RemoteRepoUrlKey, "", remoteRepoUrlDescription)
+	// If set to false, no response will be returned
+	editFlagset.Bool(EnrichResponseKey, false, "")
+	editFlagset.Bool(InteractiveKey, true, "")
+	_, err = engine.Register(WORKFLOWID_IGNORE_EDIT, workflow.ConfigurationOptionsFromFlagset(editFlagset), ignoreEditWorkflowEntryPoint)
+	if err != nil {
+		return err
+	}
+
+	deleteFlagSet := pflag.NewFlagSet(ignoreDeleteWorkflowName, pflag.ExitOnError)
+	deleteFlagSet.String(IgnoreIdKey, "", ignoreIdDescription)
+	// If set to false, no response will be returned
+	deleteFlagSet.Bool(EnrichResponseKey, false, "")
+	deleteFlagSet.Bool(InteractiveKey, true, "")
+	_, err = engine.Register(WORKFLOWID_IGNORE_DELETE, workflow.ConfigurationOptionsFromFlagset(deleteFlagSet), ignoreDeleteWorkflowEntryPoint)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func ignoreEditWorkflowEntryPoint(_ workflow.InvocationContext, _ []workflow.Data) (output []workflow.Data, err error) {
+	panic("not implemented")
+}
+
+func ignoreDeleteWorkflowEntryPoint(_ workflow.InvocationContext, _ []workflow.Data) (output []workflow.Data, err error) {
+	panic("not implemented")
+}
+
+func ignoreCreateWorkflowEntryPoint(invocationCtx workflow.InvocationContext, _ []workflow.Data) (output []workflow.Data, err error) {
+	logger := invocationCtx.GetEnhancedLogger()
+	userInterface := invocationCtx.GetUserInterface()
+	config := invocationCtx.GetConfiguration()
+	id := invocationCtx.GetWorkflowIdentifier()
+
+	interactive := config.GetBool(InteractiveKey)
+	addCreateIgnoreDefaultConfigurationValues(invocationCtx, interactive)
+
+	userName, err := getUser(invocationCtx)
+	if err != nil {
+		return nil, fmt.Errorf("user is not authenticated: %w", err)
+	}
```

Comment:
question: this seems to ignore the types of errors returned from `getUser()` and just return a not authenticated error?
suggest: implement Error Catalog errors instead of using `fmt.Error()`